packages/gatsby-source-url/src/createRootImgixImageType.ts

```diff
@@ -0,0 +1,56 @@
+import { GatsbyCache } from 'gatsby';
+import {
+  GraphQLFieldConfig,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from 'graphql';
+import type ImgixClient from 'imgix-core-js';
+import * as R from 'ramda';
+import { createImgixFixedFieldConfig } from './createImgixFixedFieldConfig';
+import { createImgixFluidFieldConfig } from './createImgixFluidFieldConfig';
+import { createImgixUrlFieldConfig } from './createImgixUrlFieldConfig';
+import { IGatsbySourceUrlRootArgs, IImgixParams } from './publicTypes';
+
+type IRootSource = {
+  rawUrl: string;
+};
+export const createRootImgixImageType = (
+  imgixClient: ImgixClient,
+  cache: GatsbyCache,
+  defaultParams: Partial<IImgixParams>,
+  // TODO: fix any
+): GraphQLFieldConfig<any, any, IGatsbySourceUrlRootArgs> => ({
+  args: {
+    url: {
+      type: GraphQLNonNull(GraphQLString),
+      description:
+        'The path of the image to render. If using a Web Proxy Source, this can also be a fully-qualified URL.',
```

Comment:
Nit: If using a Web Proxy, the path **must** be a fully-qualified URL?

Reply:
Yep, I'll update this